support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/web/OidcSecurityInterceptor.java

```diff
@@ -48,6 +55,10 @@ public boolean preHandle(final HttpServletRequest request,
         }
 
         final Set<String> prompts = authorizationRequestSupport.getOidcPromptFromAuthorizationRequest(ctx);
+        /*
```

Comment:
What is this about?

Reply:
point out the code bellow have bugs ,but I don't have a good solution for it . and maybe you will fix it. if it's useless, i will delete it.
``` java
  final Set<String> prompts = authorizationRequestSupport.getOidcPromptFromAuthorizationRequest(ctx);
        /*
         *  when prompt=login,the following code will result in a loop of authentication and this SecurityInterceptor  will return
         *  FALSE forever.
         */ 
        if (!clearCreds) {
            clearCreds = prompts.contains(OidcConstants.PROMPT_LOGIN);
        }

        if (clearCreds) {
            clearCreds = !prompts.contains(OidcConstants.PROMPT_NONE);
        }

        if (clearCreds) {
            manager.remove(true);
        }
```
when the request have parameter "prompt=login",the code above  will result in a loop of authentication。Because when callback after re-authentication ,the request still contains the parameter "prompt=login" ,and then clear the user profile ,and then re-authentication....